uk.co.compucorp.civicrm.hrcontactactionsmenu/CRM/HRContactActionsMenu/Helper/UserInformationActionGroup.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+use CRM_HRContactActionsMenu_Component_Group as ActionsGroup;
+use CRM_HRContactActionsMenu_Component_GroupButtonItem as ActionsGroupButtonItem;
+use CRM_HRContactActionsMenu_Component_UserInformationLinkItem as UserInformationLinkItem;
+use CRM_HRContactActionsMenu_Component_UserRoleItem as UserRoleItem;
+use CRM_HRCore_CMSData_PathsInterface as CMSUserPath;
+use CRM_HRCore_CMSData_UserRoleInterface as CMSUserRole;
+
+/**
+ * Class CRM_HRContactActionsMenu_Helper_UserInformationActionGroup
+ */
+class CRM_HRContactActionsMenu_Helper_UserInformationActionGroup {
+
+  /**
+   * @var array
+   */
+  private $contactUserInfo;
+  /**
+   * @var \CRM_HRCore_CMSData_PathsInterface
+   */
+  private $cmsUserPath;
+  /**
+   * @var \CRM_HRCore_CMSData_UserRoleInterface
+   */
+  private $cmsUserRole;
+
+  /**
+   * CRM_HRContactActionsMenu_Helper_UserInformationActionGroup constructor.
+   *
+   * @param array $contactUserInfo
+   *   Contact user info gotten from the contact helper
+   * @param CMSUserPath $cmsUserPath
+   * @param CMSUserRole $cmsUserRole
+   *
+   */
+  public function __construct($contactUserInfo, $cmsUserPath, $cmsUserRole) {
+    $this->contactUserInfo = $contactUserInfo;
+    $this->cmsUserRole = $cmsUserRole;
+    $this->cmsUserPath = $cmsUserPath;
+  }
+
+  /**
+   * Gets User Information Menu Group with menu items already
+   * added.
+   *
+   * @return ActionsGroup
+   */
+  public function get() {
+    $contactID = $this->contactUserInfo['contact_id'];
+    $actionsGroup = new ActionsGroup('User Information:');
+
+    if(!empty($this->contactUserInfo['cmsId'])) {
+      $userInformationLinkItem = new UserInformationLinkItem($this->cmsUserPath, $this->contactUserInfo);
+      $actionsGroup->addItem($userInformationLinkItem);
+
+      $userRoleItem = new UserRoleItem($this->cmsUserRole);
+      $actionsGroup->addItem($userRoleItem);
+
+      $actionsGroup->addItem(self::getSendWelcomeMailButton($contactID));
+      $actionsGroup->addItem(self::getSendPasswordResetButton($contactID));
+    } else {
+      $actionsGroup->addItem(self::getCreateUserButton($contactID));
```

Comment:
This class has no static methods anymore, so `self` should be replaced with `$this` for all of the 3 method calls here